.github/workflows/continuous_deployment.yml

```diff
@@ -34,7 +34,7 @@ env:
   # * rasa version in requirements.txt
   # * rasa-sdk base image version in Dockerfile
   RASA_X_VERSION: "0.40.1"
```

Comment:
We'll need to wait till we can bump this before merging since 2.8.0 is too high for 0.40.1